### DISCUSSION TITLE
Compiling error

### DISCUSSION THREAD

**Discussion by Soheilprs** [Cyfrin/foundry-full-course-cu#419] [Q&A]
when i run this commands, there was nothing happen and in skipped the compilation and stop in this section
![Screenshot from 2023-07-23 11-37-10](https://github.com/Cyfrin/foundry-full-course-f23/assets/106927616/9f94b4eb-b873-4ff3-8353-79e04593821e)
![Screenshot from 2023-07-23 11-36-57](https://github.com/Cyfrin/foundry-full-course-f23/assets/106927616/9dbbd689-253f-4a96-958e-caf1bd516c62)

**Comment by john-lawniczak** [comment]
We need more details: how long did you let it run? 

It may take some time.

**Reply by Soheilprs** [reply]
about 15 minute and it doesnt changed

**Reply by PatrickAlphaC** [reply]
what about `forge build`?

**Reply by Soheilprs** [reply]
this command is also work like this :)

**Comment by ChefAharoni** [comment]
Run `forge tree` and copy&paste it here.

Perhaps your files are in a different / inner folder?

**Reply by Soheilprs** [reply]
script/DeployFundMe.s.sol ^0.8.18
├── lib/forge-std/src/Script.sol >=0.6.2 <0.9.0
│   ├── lib/forge-std/src/console.sol >=0.4.22 <0.9.0
│   ├── lib/forge-std/src/console2.sol >=0.4.22 <0.9.0
│   ├── lib/forge-std/src/safeconsole.sol >=0.6.2 <0.9.0
│   ├── lib/forge-std/src/StdChains.sol >=0.6.2 <0.9.0
│   │   └── lib/forge-std/src/Vm.sol >=0.6.2 <0.9.0
│   ├── lib/forge-std/src/StdCheats.sol >=0.6.2 <0.9.0
│   │   ├── lib/forge-std/src/StdStorage.sol >=0.6.2 <0.9.0
│   │   │   └── lib/forge-std/src/Vm.sol >=0.6.2 <0.9.0
│   │   └── lib/forge-std/src/Vm.sol >=0.6.2 <0.9.0
│   ├── lib/forge-std/src/StdJson.sol >=0.6.0 <0.9.0
│   │   └── lib/forge-std/src/Vm.sol >=0.6.2 <0.9.0
│   ├── lib/forge-std/src/StdMath.sol >=0.6.2 <0.9.0
│   ├── lib/forge-std/src/StdStorage.sol >=0.6.2 <0.9.0 (*)
│   ├── lib/forge-std/src/StdStyle.sol >=0.4.22 <0.9.0
│   │   └── lib/forge-std/src/Vm.sol >=0.6.2 <0.9.0
│   ├── lib/forge-std/src/StdUtils.sol >=0.6.2 <0.9.0
│   │   ├── lib/forge-std/src/interfaces/IMulticall3.sol >=0.6.2 <0.9.0
│   │   └── lib/forge-std/src/Vm.sol >=0.6.2 <0.9.0
│   ├── lib/forge-std/src/Vm.sol >=0.6.2 <0.9.0
│   └── lib/forge-std/src/Base.sol >=0.6.2 <0.9.0
│       ├── lib/forge-std/src/StdStorage.sol >=0.6.2 <0.9.0 (*)
│       └── lib/forge-std/src/Vm.sol >=0.6.2 <0.9.0
├── src/FundMe.sol ^0.8.18
│   ├── lib/chainlink-brownie-contracts/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol ^0.8.0
│   └── src/PriceConverter.sol ^0.8.18
│       └── lib/chainlink-brownie-contracts/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol ^0.8.0
└── script/HelperConfig.s.sol ^0.8.18
    ├── lib/forge-std/src/Script.sol >=0.6.2 <0.9.0 (*)
    └── test/mocks/MockV3Aggregator.sol ^0.8.0
script/HelperConfig.s.sol ^0.8.18 (*)
script/Interactions.s.sol ^0.8.18
├── lib/forge-std/src/Script.sol >=0.6.2 <0.9.0 (*)
├── lib/foundry-devops/src/DevOpsTools.sol >=0.8.13 <0.9.0
│   ├── lib/forge-std/src/Vm.sol >=0.6.2 <0.9.0
│   └── lib/forge-std/src/console.sol >=0.4.22 <0.9.0
└── src/FundMe.sol ^0.8.18 (*)
src/FundMe.sol ^0.8.18 (*)
src/PriceConverter.sol ^0.8.18 (*)
test/integration/InteractionsTest.t.sol ^0.8.18
├── lib/forge-std/src/Test.sol >=0.6.2 <0.9.0
│   ├── lib/forge-std/src/console.sol >=0.4.22 <0.9.0
│   ├── lib/forge-std/src/console2.sol >=0.4.22 <0.9.0
│   ├── lib/forge-std/src/safeconsole.sol >=0.6.2 <0.9.0
│   ├── lib/forge-std/src/StdAssertions.sol >=0.6.2 <0.9.0
│   │   ├── lib/forge-std/lib/ds-test/src/test.sol >=0.5.0
│   │   └── lib/forge-std/src/StdMath.sol >=0.6.2 <0.9.0
│   ├── lib/forge-std/src/StdChains.sol >=0.6.2 <0.9.0 (*)
│   ├── lib/forge-std/src/StdCheats.sol >=0.6.2 <0.9.0 (*)
│   ├── lib/forge-std/src/StdError.sol >=0.6.2 <0.9.0
│   ├── lib/forge-std/src/StdInvariant.sol >=0.6.2 <0.9.0
│   ├── lib/forge-std/src/StdJson.sol >=0.6.0 <0.9.0 (*)
│   ├── lib/forge-std/src/StdMath.sol >=0.6.2 <0.9.0
│   ├── lib/forge-std/src/StdStorage.sol >=0.6.2 <0.9.0 (*)
│   ├── lib/forge-std/src/StdStyle.sol >=0.4.22 <0.9.0 (*)
│   ├── lib/forge-std/src/StdUtils.sol >=0.6.2 <0.9.0 (*)
│   ├── lib/forge-std/src/Vm.sol >=0.6.2 <0.9.0
│   ├── lib/forge-std/src/Base.sol >=0.6.2 <0.9.0 (*)
│   └── lib/forge-std/lib/ds-test/src/test.sol >=0.5.0
├── src/FundMe.sol ^0.8.18 (*)
├── script/DeployFundMe.s.sol ^0.8.18 (*)
└── script/Interactions.s.sol ^0.8.18 (*)
test/mocks/MockV3Aggregator.sol ^0.8.0
test/unit/FundMeTest.t.sol ^0.8.18
├── lib/forge-std/src/Test.sol >=0.6.2 <0.9.0 (*)
├── src/FundMe.sol ^0.8.18 (*)
└── script/DeployFundMe.s.sol ^0.8.18 (*)

**Reply by ChefAharoni** [reply]
Try two things:

1. Make an error on purpose in your FundMe.sol file, and then try to compile it using `forge build`.

2. Paste your script and test files so we'll see if there's something wrong with your deployment.
